### PR TITLE
Fix HTML response during redirection

### PR DESCRIPTION
**Pull request by datval**
Fixes issue #11370 by fixing response HTML

### CODE REVIEW

File: src/main/java/org/elasticsearch/http/HttpServer.java [hunk 1]
```diff
@@ -155,7 +155,7 @@ void handlePluginSite(HttpRequest request, HttpChannel channel) throws IOExcepti
             sitePath = null;
             // If a trailing / is missing, we redirect to the right page #2654
             String redirectUrl = request.rawPath() + "/";
-            BytesRestResponse restResponse = new BytesRestResponse(RestStatus.MOVED_PERMANENTLY, "text/html", "<head><meta http-equiv=\"refresh\" content=\"0; URL=" + redirectUrl + "></head>");
+            BytesRestResponse restResponse = new BytesRestResponse(RestStatus.MOVED_PERMANENTLY, "text/html", "<head><meta http-equiv=\"refresh\" content=\"0; ,URL=" + redirectUrl + "\"></head>");
```
Comment (clintongormley) [inline src/main/java/org/elasticsearch/http/HttpServer.java]: You have an extra `,` before the URL

**Comment by clintongormley** [comment]
Hi @datval 

Thanks for the PR.  Please could you remove the extra comma that has been added, and sign the CLA so that we can merge this in: http://www.elasticsearch.org/contributor-agreement/

thanks

**Comment by datval** [comment]
Thanks @clintongormley, I just signed the CLA, I guess it needs some time to reflect here.

**Comment by clintongormley** [comment]
@datval did you sign the corporate CLA?  I ask because those have to be checked manually.  I don't see you in the personal CLA list.

**Comment by datval** [comment]
@clintongormley Yep, I signed it, but as "An Individual contributing under an existing company contributor agreement".

**Comment by clintongormley** [comment]
@datval thanks - I'll need to look it up manually

**Comment by datval** [comment]
@clintongormley Do you want reference number?

**Comment by clintongormley** [comment]
@datval got it, thanks!

**Comment by bleskes** [comment]
LGTM. Will merge soon :)

**Pull request closed without merging**